core/templates/dev/head/components/top_navigation_bar/top_navigation_bar_directive.html

```diff
@@ -125,7 +125,7 @@
         </li>
         <li>
           <a ng-if="(isAdmin || isTopicManager) && newStructuresEnabled" ng-click="closeSubmenuIfNotMobile($event)" href="/topics_and_skills_dashboard"
-             class="protractor-test-dashboard-link">
+             class="protractor-test-dashboard-link protractor-test-topics-and-skills-dashboard-link">
```

Comment:
Do you need the protractor-test-dashboard-link class too? If that is used for styling, and not as a selector for protractor, consider dropping protractor- from the name.

Reply:
For the Creator Dashboard and Learner Dashboard links in the dropdown also, 2 protractor-test- classes are there, so I just followed the same format. It seems like they aren't used anywhere now, shall I remove all 3 instances then?

Reply:
Done, removed `protractor-test-dashboard-link` from all 3 classes as they were not used anywhere else.